samples/proto-column-mutation-insert.js

```diff
@@ -0,0 +1,101 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+// eslint-disable-next-line node/no-unpublished-require
+const singer = require('./resource/singer.js');
+const music = singer.spanner.examples.music;
+
+function main(
+  instanceId = 'my-instance',
+  databaseId = 'my-database',
+  projectId = 'my-project-id'
+) {
+  // [START spanner_insert_proto_columns_data]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  // const instanceId = 'my-instance';
+  // const databaseId = 'my-database';
+  // const projectId = 'my-project-id';
+
+  // Imports the Google Cloud Spanner client library
+  const {Spanner} = require('@google-cloud/spanner');
+
+  // Instantiates a client
+  const spanner = new Spanner({
+    projectId: projectId,
+  });
+
+  async function protoColumnMutationInsert() {
+    // Gets a reference to a Cloud Spanner instance and database.
+    const instance = spanner.instance(instanceId);
+    const database = instance.database(databaseId);
+
+    const genre = music.Genre.ROCK;
+    const singerInfo = music.SingerInfo.create({
+      singerId: 1,
+      genre: genre,
+      birthDate: 'January',
+      nationality: 'Country1',
+    });
+
+    const protoMessage = Spanner.protoMessage({
+      value: singerInfo,
+      messageFunction: music.SingerInfo,
+      fullName: 'spanner.examples.music.SingerInfo',
+    });
+
+    const protoEnum = Spanner.protoEnum({
+      value: genre,
+      enumObject: music.Genre,
+      fullName: 'spanner.examples.music.Genre',
+    });
+
+    // Instantiate Spanner table object.
```

Comment:
'Instantiate' sounds like we are creating it

```suggestion
    // Get a reference to the Singers table
```